Client/Application/AppDelegate.swift

```diff
@@ -179,6 +201,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // We sync in the foreground only, to avoid the possibility of runaway resource usage.
     // Eventually we'll sync in response to notifications.
     func applicationDidBecomeActive(application: UIApplication) {
+        // handle quick actions is available
+        if #available(iOS 9, *) {
+            guard let shortcut = quickActions.launchedShortcutItem else { return }
+            quickActions.handleShortCutItem(shortcut, completionBlock: handleShortCutItemType)
+            quickActions.launchedShortcutItem = nil
```

Comment:
No, but there is an error there. We are returning when there is no launchedShortcutItem and never executing the  `self.profile?.syncManager.applicationDidBecomeActive()` & `self.browserViewController.loadQueuedTabs()`, which means that running iOS9 we will never execute these lines unless we have opened by a shortcut. I'll turn the `guard` into an `if let`. I think we want to execute these lines regardless, for now. If we need to prevent passthrough later we can amend, but right now I can't think of a reason why we would.